### ISSUE TITLE
Not an issue but a thanks + question !

### ISSUE THREAD

**Issue by 2Belette** [lukaspieper/Gcam-Services-Provider#57]
First, many thanks! you make my experience far better! 
I just got a brand new OnePlus 9T Pro and just installed Lineageos 18.1 as I don't want any Google like app.
I have to make an expection with GCam 8.1 to enjoy the powerful cameras, as with the stock Lineageos camera app it is very limited...
You are the one who make it possible as Lineageos doesn't come with Gapps and I didn't want to go with MicroG as I had it before (coming from /e/) so thanks again and if you have donation page, please share it :)

Just a quick question regarding privacy, it seems to me that everything is locally emulated so I guess Gcam is not initiating any connection to any Gapps services ? I will try to retrieve some logs/packet capture to confirm but just wanted to have some perspective before.
Maybe something included into Gcam is trying to initiate some connections anyway to other services/components apart from Gapps services.

**Comment by lukaspieper** [comment]
Hi,

> Just a quick question regarding privacy, it seems to me that everything is locally emulated so I guess Gcam is not initiating any connection to any Gapps services ? [...] Maybe something included into Gcam is trying to initiate some connections anyway to other services/components apart from Gapps services.

The whole thing is quite complex. To state the obvious, my app has no internet permission, does not store any data and does not do anything else that would be relevant in terms of data protection/privacy. I have only faked the absolutely necessary interfaces. This does not mean that Gcam cannot try to access other Play Services. On a system without Play Services, however, this comes to nothing. 

What Gcam will probably do, however, is communicate directly with Google's servers, as the camera app has declared internet permission. However, since the OS I am using can deny this permission, I have not looked into this further. Preventing conntections would not be within the possibilities of this project.

tl;dr If you have my app installed and you can make sure that Gcam has no Internet access, you are good to go.

>  if you have donation page, please share it :)

Thank you for the implicit offer, I appreciate it. So far I have not accepted any donations because I am very unsure about the legal situation. Also, this project would not have been possible in its current state without microg and Calyx. I have only offered the Gcam relevant parts individually, so I have not done any pioneering work.

However, I will revisit the topic of donations as soon as I publish my next project. I've put a lot more time into it and I hope that it will find a lot of interest.
